matching-notification-ui/src/main/resources/MatchingNotification/MatchesTable.xml

```diff
@@ -0,0 +1,568 @@
+        &lt;th data-column="matchedEmails" title="$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.titles.matchedEmails'))"&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.headings.matched'))&lt;span class="fa fa-angle-double-left collapse-marker"&gt;&lt;/span&gt;&lt;/th&gt;
+        &lt;th data-column="notified" title="$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.titles.notified'))"&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.headings.notified'))&lt;/th&gt;
+      &lt;/tr&gt;
+      &lt;/thead&gt;
+      &lt;tbody&gt;
+      &lt;/tbody&gt;
+    &lt;/table&gt;
+
+## Pagination
+    &lt;div class="xwiki-livetable-pagination" id="pagination-matching-notifications"&gt;
+      &lt;span class="controlPagination"&gt;
+        &lt;a title="$escapetool.xml($services.localization.render('platform.livetable.paginationPagePrevTitle'))" class="prevPagination" id="pagination-prev" href="#"&gt;&lt;span class="hidden"&gt;$services.localization.render('platform.livetable.paginationPagePrevTitle')&lt;/span&gt;&lt;/a&gt;
+        &lt;a title="$escapetool.xml($services.localization.render('platform.livetable.paginationPageNextTitle'))" class="nextPagination" id="pagination-next" href="#"&gt;&lt;span class="hidden"&gt;$services.localization.render('platform.livetable.paginationPageNextTitle')&lt;/span&gt;&lt;/a&gt;
+      &lt;/span&gt;
+      &lt;span class="pagination"&gt;
+        &lt;span class="xwiki-livetable-pagination-text"&gt;$services.localization.render('platform.livetable.paginationPage')&lt;/span&gt;
+        &lt;span class="xwiki-livetable-pagination-content" id="pagination-content"&gt;&lt;/span&gt;
+      &lt;/span&gt;
+    &lt;/div&gt;
+##--SEND NOTIFICATION BUTTON
+    &lt;br/&gt;
+    &lt;div #if(!$hasAdmin)class="hidden"#end&gt;
+      &lt;span class="buttonwrapper"&gt;
+        &lt;a class="button" id="send-notifications-button" href="#"&gt;&lt;span class="fa fa-envelope" &gt;&lt;/span&gt;
+          &lt;span&gt;$escapetool.xml($services.localization.render('phenotips.matchingNotifications.table.notify.label'))&lt;/span&gt;
+        &lt;/a&gt;
+      &lt;/span&gt;
+      &lt;div id="send-notifications-messages"&gt;&lt;/div&gt;
+    &lt;/div&gt;
+    &lt;/div&gt;##--closing table div id="table-container"
+  &lt;/div&gt;
+&lt;/div&gt;
+{{/html}}
+{{/velocity}}</content>
+<object>
+    <name>Families.WebHome</name>
+    <number>0</number>
+    <className>XWiki.UIExtensionClass</className>
+    <guid>e0686a4d-61ec-4d99-9941-fb0041842158</guid>
```

Comment:
Don't just copy GUIDs, they should be unique. Same for the other objects.

Reply:
we will remove this object and replace with a WebHome for this space